src/main/java/tc/oc/pgm/commands/DiscordCommands.java

```diff
@@ -0,0 +1,83 @@
+package tc.oc.pgm.commands;
+
+import app.ashcon.intake.Command;
+import app.ashcon.intake.argument.ArgumentException;
+import java.util.Map;
+import java.util.UUID;
+import javax.annotation.Nullable;
+import net.dv8tion.jda.api.entities.User;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import tc.oc.pgm.AllTranslations;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.discord.DiscordClient;
+
+public class DiscordCommands {
+
+  @Command(
+      aliases = {"discord"},
+      desc = "Discord Integration Commands",
+      usage = "[action] <option>")
+  public static void discord(
+      CommandSender sender, MatchPlayer player, String action, @Nullable String option)
+      throws ArgumentException {
+    if (player == null)
+      throw new ArgumentException(AllTranslations.get().translate("command.onlyPlayers", sender));
+    switch (action) {
+      case "link":
+        User user = DiscordClient.getUserFromTag(option);
+        // TODO: Check if user has linked an account
+        UUID uuid = player.getId();
+        if (DiscordClient.TOKENS_CACHE.getIfPresent(uuid) == null) {
```

Comment:
I would create methods in `DiscordClient` to abstract this stuff so that other modules can also link/unlink. It will also made this command method cleaner